mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/ClassLoaderHandle.java

```diff
@@ -13,8 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.mantisrx.server.worker;
+package io.mantisrx.server.master.client;
```

Comment:
Honestly, I'm not sure if this is the right package for this class. mantis-control-plane-client is supposed to hold all the stuff relating to talking to the mantis-control-plane. So, I'm not sure if this code belongs there. 

I would put this under mantis/taskexecutor/api or mantis/server/worker/client.